docs/src/pages/components/data-grid/export/export.md

```diff
@@ -6,23 +6,40 @@ title: Data Grid - Export
 
 <p class="description">Easily export the rows in various file formats such as CSV, Excel, or PDF.</p>
 
-## CSV export
+## Enabling export
+
+### Default Toolbar
 
-The easiest way to enable the CSV export is to pass the `GridToolbar` component in the `Toolbar` [component slot](/components/data-grid/components/#toolbar).
+To enable the export selector, pass the `GridToolbar` component in the `Toolbar` [component slot](/components/data-grid/components/#toolbar).
```

Comment:
```suggestion
The easiest way to enable the export selector is to pass the `GridToolbar` component in the `Toolbar` [component slot](/components/data-grid/components/#toolbar).
```

Or even keep the sentence non technical and let the example explain the implementation details

```suggestion
The easiest way to enable the export selector is to enable the toolbar.
```

Reply:
A tech writing rule from GitLab dev-rel is to avoid words such as "easy": 
> Don't say it's easy because it might not be for readers, just explain how to do it.

Reply:
@joserodolfofreitas is using it
But if it is advised not to

Reply:
If it interests someone, here is the 4 tutorial video
https://about.gitlab.com/handbook/engineering/ux/technical-writing/fundamentals/

Reply:
Right, it makes sense. I'll be attentive when affirming something is easy. 
Although in this case, it's saying it's easiest compared with the other options, which is arguably not the same.
Said like that, I don't think it's patronizing or subjective as "easy" can be. I'd say it can be an important piece of information to devs, if they're looking for the quickest/easiest way of using the feature.
But I'm fine not using it.